simple_https_client/src/tls13client.rs

```diff
@@ -1,11 +1,40 @@
-use std::{env, net::TcpStream, str::FromStr};
+use std::net::TcpStream;
 
-use anyhow::Context;
 use bertie::tls13utils::*;
 use record::AppError;
-use simple_https_client::{ciphersuites, tls13client};
+use simple_https_client::{ciphersuite_from_str, tls13client};
 use tracing::{error, trace};
 
+mod flags {
+    xflags::xflags! {
```

Comment:
Hm, this looks weird. I'm all for making a better cli, but let's use clap.
Or do you have a reason do use something else?

Reply:
Yeah, I've changed it to use `clap` :+1: